src/conftest.py

```diff
@@ -11,10 +13,19 @@
 import config
 from analysis.PluginBase import AnalysisBasePlugin
 from config import Config
-from test.common_helper import CommonDatabaseMock, create_docker_mount_base_dir
+from test.common_helper import CommonDatabaseMock
 from test.conftest import merge_markers
 
 
+def _create_docker_mount_base_dir() -> str:
+    dir = tempfile.mkdtemp(prefix='fact-docker-mount-base-dir')
```

Comment:
`dir` is a python builtin and should not be used

```suggestion
    tmp_dir = tempfile.mkdtemp(prefix='fact-docker-mount-base-dir')
```